[2024-11-17T18:00:20.418+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-17T18:00:20.446+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: add_data_API.run_API_news scheduled__2024-11-17T12:00:00+00:00 [queued]>
[2024-11-17T18:00:20.459+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: add_data_API.run_API_news scheduled__2024-11-17T12:00:00+00:00 [queued]>
[2024-11-17T18:00:20.460+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-17T18:00:20.483+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): run_API_news> on 2024-11-17 12:00:00+00:00
[2024-11-17T18:00:20.498+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1418) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-17T18:00:20.500+0000] {standard_task_runner.py:72} INFO - Started process 1422 to run task
[2024-11-17T18:00:20.500+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'add_data_API', 'run_API_news', 'scheduled__2024-11-17T12:00:00+00:00', '--job-id', '698', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpcauk63ch']
[2024-11-17T18:00:20.502+0000] {standard_task_runner.py:105} INFO - Job 698: Subtask run_API_news
[2024-11-17T18:00:20.563+0000] {task_command.py:467} INFO - Running <TaskInstance: add_data_API.run_API_news scheduled__2024-11-17T12:00:00+00:00 [running]> on host a4a76eee4b32
[2024-11-17T18:00:20.712+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='add_data_API' AIRFLOW_CTX_TASK_ID='run_API_news' AIRFLOW_CTX_EXECUTION_DATE='2024-11-17T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-17T12:00:00+00:00'
[2024-11-17T18:00:20.714+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-17T18:00:20.715+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-17T18:00:20.716+0000] {logging_mixin.py:190} INFO - Current task name:run_API_news state:running start_date:2024-11-17 18:00:20.447489+00:00
[2024-11-17T18:00:20.716+0000] {logging_mixin.py:190} INFO - Dag name:add_data_API and current dag run status:running
[2024-11-17T18:00:20.718+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-17T18:00:20.824+0000] {logging_mixin.py:190} INFO - Erreur lors de l'ex√©cution du script : Command '['python', 'C:/Users/33672/OneDrive/Documents/GitHub/Projet_EDD/script/getAPI.py']' returned non-zero exit status 2.
[2024-11-17T18:00:20.825+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-17T18:00:20.849+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-17T18:00:20.850+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=add_data_API, task_id=run_API_news, run_id=scheduled__2024-11-17T12:00:00+00:00, execution_date=20241117T120000, start_date=20241117T180020, end_date=20241117T180020
[2024-11-17T18:00:20.872+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-17T18:00:20.874+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-17T18:00:20.875+0000] {logging_mixin.py:190} INFO - Dag name:add_data_API queued_at:2024-11-17 18:00:01.250738+00:00
[2024-11-17T18:00:20.877+0000] {logging_mixin.py:190} INFO - Task hostname:a4a76eee4b32 operator:PythonOperator
[2024-11-17T18:00:20.920+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-17T18:00:20.941+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
